cellpack/autopack/Analysis.py

```diff
@@ -828,9 +828,12 @@ def update_distance_distribution_dictionaries(
 
         get_angles = False
         if ingr.packing_mode == "gradient" and self.env.use_gradient:
-            self.center = center = self.env.gradients[ingr.gradient].mode_settings.get(
-                "center", center
-            )
+            if not isinstance(ingr.gradient, list):
+                self.center = center = self.env.gradients[
+                    ingr.gradient
+                ].mode_settings.get("center", center)
+            else:
+                self.center = center
```

Comment:
nit: maybe simply reverse `if` and `else` statements for readability?
```suggestion
            if isinstance(ingr.gradient, list):
                self.center = center
            else:
                self.center = self.env.gradients[
                    ingr.gradient
                ].mode_settings.get("center", center)
```

Reply:
Thanks Ruge! This reminded me of another edge case of a single gradient in a list so I added that in as well [here](https://github.com/mesoscope/cellpack/pull/270/commits/5614cd0285d7bd401f0c94d0a56efd73a0bf1265).